tests/model_tests/test_middleware_auth.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Group
+from guardian.shortcuts import assign_perm
+import pytest
+# Allow everything in there to access the DB
+pytestmark = pytest.mark.django_db
+
+import logging
+
+from nsot import exc, models
+from nsot.middleware.auth import NsotObjectPermissionsBackend
+
+from .fixtures import admin_user, user, site, transactional_db
+
+def test_object_level_permissions_with_ancestors(site, user):
+    net_8  = models.Network.objects.create(site=site, cidr=u'8.0.0.0/8')
+    net_24 = models.Network.objects.create(site=site, cidr=u'8.0.0.0/24')
+    net_16 = models.Network.objects.create(site=site, cidr=u'8.0.0.0/16')
+
+    # Need to refresh the objects from the db so the updated parent_ids are
+    # reflected.
+    net_8.refresh_from_db()
+    net_24.refresh_from_db()
+    net_16.refresh_from_db()
+
+    grp = Group.objects.create(name='test_group')
```

Comment:
If `test_group` is used in any other tests, you may want to consider making it a fixture to ensure it gets properly created and torn down for every test case.

Reply:
Done.